vizro-core/mkdocs.yml

```diff
@@ -93,7 +94,11 @@ markdown_extensions:
       generic: true
   - footnotes
   - pymdownx.details
-  - pymdownx.superfences
+  - pymdownx.superfences:
```

Comment:
What doest this do?

Reply:
It enables the mermaid diagram like this:

````
``` mermaid
```
````